conf/Config.js

```diff
@@ -215,6 +216,18 @@ class Config extends EventEmitter {
     getIsTransientLocation(locationName) {
         return this.transientLocations[locationName] || false;
     }
+
+    getInstanceId() {
+        return this.instanceId;
+    }
+
+    setInstanceId(instanceId) {
```

Comment:
This should be coming from the Cloudserver Manager. It should never be set by an Backbeat process.